### PR TITLE
[Bug] Improve alignment of top bar labels , text hover effect #273

### PR DESCRIPTION
**Pull request by Sai-Manikanta-Andey**
### Overview
This Pull Request addresses issue #273. It includes improvements to the alignment of top bar labels and introduces text hover effect on tabs.

### Details
- Fixed the alignment issues with top bar labels to ensure they are properly positioned , add text hover effect on tabs.

### What is the purpose of this pull request?
- [ ] New feature
- [ ] Documentation update
- [x] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other: Spelling corrections

### CODE REVIEW

File: app/src/routes/project/LayersPanel/index.tsx [hunk 1]
```diff
@@ -22,15 +22,21 @@ const LayersPanel = observer(() => {
...
+                    <TabsTrigger
+                        className="bg-transparent p-0 text-xs hover:text-white"
+                        value={TabValue.COMPONENTS}
+                    >
                         {capitalizeFirstLetter(TabValue.COMPONENTS)}
                     </TabsTrigger>
                     <div className="flex-grow"></div>
                     <button
-                        className="text-white hover:text-text w-6 h-6"
+                        className="text-white hover:text-text w-6 h-6 pt-[3]"
```
Comment (Kitenite) [inline app/src/routes/project/LayersPanel/index.tsx:39]: Just double checking, is `pt-[3]` is this supposed to be 3px?

Comment (Sai-Manikanta-Andey) [reply app/src/routes/project/LayersPanel/index.tsx:39]: hey @Kitenite  yes it should be 3px maybe i made a mistake,i have actually put 3px only maybe somehow i changed it , let me fix it quickly !

**Review by drfarrell — APPROVED** [review]
Seems good

File: app/src/components/Announcement/index.tsx [hunk 2]
```diff
@@ -103,19 +103,19 @@ function Announcement() {
                     <div className="grid grid-cols-2">
                         <div className="space-y-2">
                             <p>Resources</p>
-                            <div className="flex flex-col -space-y-1 ml-2 items-start">
+                            <div className="flex flex-col items-start">
                                 <Button
                                     variant="link"
                                     size="sm"
-                                    className="text-text flex flex-row items-center"
+                                    className="text-text flex flex-row items-center hover:text-text-active"
```
Comment (Kitenite) [inline app/src/components/Announcement/index.tsx:110]: Might be better to update the link variant in button but this works too

**Comment by Kitenite** [comment]
Looks great to me, awesome work @Sai-Manikanta-Andey @drfarrell !

**Pull request merged**